sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/BasicWriteStatsTracker.scala

```diff
@@ -192,6 +192,10 @@ class BasicWriteJobStatsTracker(
     new BasicWriteTaskStatsTracker(serializableHadoopConf.value)
   }
 
+  override def processCommitDuration(duration: Long): Unit = {
+    metrics(BasicWriteJobStatsTracker.DURATION_FILE_COMMIT).set(duration)
+  }
+
   override def processStats(stats: Seq[WriteTaskStats]): Unit = {
```

Comment:
Instead of adding a new method, we can probably just add one more parameter here. Then we don't need to worry about the execution order of `processCommitDuration` and `processStats`.

Reply:
> Instead of adding a new method, we can probably just add one more parameter here. Then we don't need to worry about the execution order of `processCommitDuration` and `processStats`.

Done